docs/terminal/eastsale.md

```diff
@@ -4,7 +4,226 @@
 
 --8<-- "includes/abbreviations.md"
 
-Reserved.
+## Positions
 
-!!! note
-    Additional charts to the AIP may be found in the RAAF TERMA document, available towards the bottom of [RAAF AIP page](https://ais-af.airforce.gov.au/australian-aip){target=new}
\ No newline at end of file
+| Name               | ID      | Callsign             | Frequency        | Login Identifier    |
+| ------------------ | ------- | ------------------- | ---------------- | ------------------- |
+| **East Sale TMA**  | **ESA** | **East Sale Approach** | **123.300**      | **ES_APP**          |
+
+## Airspace
+
+ES TCU controls the airspace within R360. Typically, sectors R360A through R360F are active. The enroute controllers manage the airspace beneath the active R360 sectors, with YWE controlling the southern section and ELW controlling the northern section.
```

Comment:
Just confirming enroute controllers manage the airspace *beneath* R360? I'd be happy to give ESA ownership of the class G below the PRD, for simplicity. Thoughts @alphadelta332 ?

Reply:
IRL Melbourne Centre owns the Class G beneath. Keeps it simple I think because then ES APP just owns the Restricted Areas, nothing else, which is in line with other Military TCUs too. There's pros and cons both ways I think, But I'd give a slight edge to Melbourne Centre owning the underlying Class G personally

Reply:
Yeah ok, I was just trying to avoid a triple vertical split which might get confusing without the diagram in front of you. Happy to leave as is